docs/AboutUs.md

```diff
@@ -5,55 +5,16 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](https://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `e1290028@u.nus.edu`
 
 ## Project team
 
-### John Doe
+### Leonardo Wolf
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/leowolf275.png" width="200px">
```

Comment:
I found that using local paths causes a bug that doesn't let the image render, so maybe we should use the generated asset link: https://ay2425s1-cs2103t-w14-4.github.io/tp/images/leowolf275.png ?

Also maybe we should add alt text for accessibility or see whether there's an issue in the image rendering? Here's an example:
`<img src="https://ay2425s1-cs2103t-w14-4.github.io/tp/images/youngseopark05.png" alt="Youngseo Park's picture" width="200px" />`

Reply:
Thank you!

Reply:
Fixed!